src/scripts/content/trello.js

```diff
@@ -47,18 +49,25 @@ togglbutton.render(
   '.checklist-item-details:not(.toggl)',
   { observe: true },
   (elem) => {
-    const titleElem = $('.window-title h2');
-    const taskElem = $('.checklist-item-details-text', elem);
+    const getTitleText = () => {
+      const title = $('.window-title h2', elem);
```

Comment:
`elem` here is checklist item, without `h2` tag inside 🤔 
therefore it always result with empty string